zipkin-server/pom.xml

```diff
@@ -283,6 +283,7 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
         <configuration>
+          <layout>ZIP</layout>
```

Comment:
Add a comment that this forces Spring Boot to use the `PropertiesLauncher` for startup, enabling `loader.path`.